packages/flutter/lib/src/widgets/focus_manager.dart

```diff
@@ -395,6 +416,374 @@ class FocusScopeNode extends Object with DiagnosticableTreeMixin {
   }
 }
 
+typedef FocusableNodeVisitor = bool Function(FocusableNode node);
+
+/// An interface for data from policy objects to be held by the scope nodes.
+///
+/// This data is set and requested via [FocusableNode.policyData].
+///
+/// If a policy data object encounters a type of policy data that it doesn't
+/// recognize, then it should not use it, but may overwrite it (this may happen
+/// when the traversal policy is changed).
+abstract class PolicyData {
+  /// A const constructor so that subclasses can be const.
+  const PolicyData();
+}
+
+/// A node used by [Focusable] to represent and change the input focus.
+///
+/// A `FocusableNode` is created and managed by the [Focusable] widget, forming
```

Comment:
use [] rather than \`\` even for self-references, it makes copy-and-paste later more resilient and will make it easier to catch errors later once we have lints for `[]` being to actual identifiers.